makefile

```diff
@@ -42,27 +42,40 @@ LIBS := -lm
 # multithreading for GCC: C++/C/Fortran, MATLAB, and octave (ICC differs)...
 OMPFLAGS = -fopenmp
 OMPLIBS = -lgomp
-MOMPFLAGS = -D_OPENMP
+# we bundle any libs mex needs here with flags...
+MOMPFLAGS = -D_OPENMP $(OMPLIBS)
 OOMPFLAGS =
 # MATLAB MEX compilation (also see below +=)...
-MFLAGS := -largeArrayDims
+MFLAGS := -DR2008OO -largeArrayDims
 # location of MATLAB's mex compiler (could add flags to switch GCC, etc)...
 MEX = mex
 # octave, and its mkoctfile and flags (also see below +=)...
 OCTAVE = octave
 MKOCTFILE = mkoctfile
-OFLAGS =
+OFLAGS = -DR2008OO
 # For experts only, location of MWrap executable (see docs/install.rst):
 MWRAP = mwrap
 
-# dependency root (relative to top directory)
+# root directory for dependencies to be downloaded:
 DEPS_ROOT := deps
 
-# xsimd dependency repo URL
+# xsimd header-only dependency repo
 XSIMD_URL := https://github.com/xtensor-stack/xsimd.git
 XSIMD_VERSION := 13.0.0
 XSIMD_DIR := $(DEPS_ROOT)/xsimd
 
+# DUCC sources optional dependency repo
+DUCC_URL := https://gitlab.mpcdf.mpg.de/mtr/ducc.git
+DUCC_VERSION := ducc0_0_34_0
```

Comment:
should not be the latest checkout otherwise does not work on mac?

Reply:
will get fixed when I loosen test/finufft?d_test exit codes.